testing/docker/elasticsearch/roles.yml

```diff
@@ -1,8 +1,12 @@
 apm_server:
-  cluster: ["manage_ilm"]
+  cluster: ['manage_ilm','manage_security','manage_api_key']
   indices:
     - names: ['apm-*']
       privileges: ['write','create_index','manage','manage_ilm']
+  applications:
```

Comment:
why this?

Reply:
This role is applied to the `apm_server_user`, which with these changes has the privileges necessary for API Key creation, so that this user can be used for the API Key tests, instead of the superuser.

Reply:
IIUC all needed for `apm_server_user` to create API keys are the `cluster` privileges added above, the `applications` section is not needed.

Reply:
ah, scratch that. it is